### PR TITLE
add some guidance about RFCs, update for reality, prep for scope definition

### PR DESCRIPTION
**Pull request by btm**
I realized listing a bunch of projects that fall under the chef-rfc umbrella would in fact look a lot like a list and should probably be a special low-numbered RFC.

### CODE REVIEW

File: rfc000-rfc-process.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ community use RFCs to document, discuss, and plan improvements to Chef and
 other Chef ecosystem tools. You can always find the list of accepted RFCs at
 https://github.com/chef/chef-rfc.
 
+The specific projects covered by this process are detailed in RFC002 (Scope).
```
Comment (coderanger) [inline rfc000-rfc-process.md]: I assume you are writing this now?

Comment (btm) [inline rfc000-rfc-process.md]: next, now is :beer:. 

This didn't feel like the right document for a long list, especially once I started pondering grouping by exceptions. For instance chef-server and test-kitchen have their own maintainers file. Most the RFCs here are applicable to chef-server, but not so much test-kitchen. However I don't want more than one chef-rfc either.

Obviously early feedback welcome.

**Comment by thommay** [comment]
The additional wording around process is much better, thanks!

**Comment by nathenharvey** [comment]
👍

**Comment by lamont-granquist** [comment]
👍

**Comment by thommay** [comment]
I decide this, and also i merge this!

**Pull request merged**